src/main/kotlin/io/snyk/plugin/ui/jcef/OpenFileLoadHandlerGenerator.kt

```diff
@@ -35,32 +35,76 @@ class OpenFileLoadHandlerGenerator(snykCodeFile: SnykCodeFile) {
         return JBCefJSQuery.Response("success")
     }
 
+    fun isDarcula(): Boolean {
+        val scheme = EditorColorsManager.getInstance().globalScheme
+        return scheme.name.contains("Darcula")
+    }
+
```

Comment:
Why do we need to specify specific themes? There are thousands on the marketplace, is there a chance to have generic handling?

Reply:
I'll get back to you in a moment 🙏

Reply:
@neilm-snyk , there is a follow-up ticket to address the styles from JetBrains APIs, so I believe it will automatically pick up the colours, fonts, and so on. Thank you 🙏